drivers/adc/ad7193/ad7193.h

```diff
@@ -175,6 +175,9 @@
 #define AD7193_GPOCON_P1DAT     (1 << 1) // P1 state
 #define AD7193_GPOCON_P0DAT     (1 << 0) // P0 state
 
+/* Channel Mask */
+#define AD7193_CH_MASK(x)		(1 << x)
```

Comment:
I think we could use the [BIT](https://github.com/analogdevicesinc/no-OS/blob/master/include/util.h) macro here.

Reply:
Will use it here.